road/config.ini

```diff
@@ -1,32 +1,35 @@
-[DEFAULT]
-DBF_Files = LOADAM_FINAL.dbf, LOADMD_FINAL.dbf, LOADPM_FINAL.dbf ,LOADEV_FINAL.dbf, LOADEA_FINAL.dbf
+[DBF]

```

Comment:
name these as loaded_network_files (@sdrewc / @dantischler is this accurate?) instead of dbf_files
(in general, when possible, we want to say what the file is about, rather than the format (unless if the format is important)